Meta/CMake/time_zone_data.cmake

```
@@ -38,30 +38,15 @@ set(TZDB_SOUTH_AMERICA_PATH "${TZDB_PATH}/${TZDB_SOUTH_AMERICA_SOURCE}")
 set(TZDB_ZONE_1970_SOURCE zone1970.tab)
 set(TZDB_ZONE_1970_PATH "${TZDB_PATH}/${TZDB_ZONE_1970_SOURCE}")
 
-function(extract_tzdb_file source path)
-    if(EXISTS "${TZDB_ZIP_PATH}" AND NOT EXISTS "${path}")
-        message(STATUS "Extracting TZDB ${source} from ${TZDB_ZIP_PATH}...")
-        execute_process(COMMAND "${TAR_TOOL}" -C "${TZDB_PATH}" -xzf "${TZDB_ZIP_PATH}" "${source}" RESULT_VARIABLE tar_result)
-        if (NOT tar_result EQUAL 0)
-            message(FATAL_ERROR "Failed to unzip ${source} from ${TZDB_ZIP_PATH} with status ${tar_result}")
-        endif()
-    endif()
-endfunction()
-
 if (ENABLE_TIME_ZONE_DATABASE_DOWNLOAD)
     remove_path_if_version_changed("${TZDB_VERSION}" "${TZDB_VERSION_FILE}" "${TZDB_PATH}")
 
     download_file("${TZDB_ZIP_URL}" "${TZDB_ZIP_PATH}")
-    extract_tzdb_file("${TZDB_AFRICA_SOURCE}" "${TZDB_AFRICA_PATH}")
-    extract_tzdb_file("${TZDB_ANTARCTICA_SOURCE}" "${TZDB_ANTARCTICA_PATH}")
-    extract_tzdb_file("${TZDB_ASIA_SOURCE}" "${TZDB_ASIA_PATH}")
-    extract_tzdb_file("${TZDB_AUSTRALASIA_SOURCE}" "${TZDB_AUSTRALASIA_PATH}")
-    extract_tzdb_file("${TZDB_BACKWARD_SOURCE}" "${TZDB_BACKWARD_PATH}")
-    extract_tzdb_file("${TZDB_ETCETERA_SOURCE}" "${TZDB_ETCETERA_PATH}")
-    extract_tzdb_file("${TZDB_EUROPE_SOURCE}" "${TZDB_EUROPE_PATH}")
-    extract_tzdb_file("${TZDB_NORTH_AMERICA_SOURCE}" "${TZDB_NORTH_AMERICA_PATH}")
-    extract_tzdb_file("${TZDB_SOUTH_AMERICA_SOURCE}" "${TZDB_SOUTH_AMERICA_PATH}")
-    extract_tzdb_file("${TZDB_ZONE_1970_SOURCE}" "${TZDB_ZONE_1970_PATH}")
+    message(STATUS "Extractintg ${TZDB_ZIP_PATH} into ${TZDB_PATH}")
```

```suggestion
    message(STATUS "Extracting ${TZDB_ZIP_PATH} into ${TZDB_PATH}")
```
Same in the other file

---

Done, thanks! (and all the other hints you gave me in the discord channel!)